### PR TITLE
[IMP] board,web: cleaner layout and kanban/pivot integration on mobile

### PR DESCRIPTION
**Pull request by res-odoo**
This merge improves the Dashboard app and make it more usable on mobile:
- The layout is fixed to 1 column.
- Embedded Kanban view is fitted to its content and handle correctly empty columns.
- Embedded Pivot view has smaller left indentation for readability purpose.

Task ID: 1904747

### CODE REVIEW

File: addons/web/static/src/scss/kanban_view_mobile.scss [hunk 1]
```diff
@@ -38,7 +38,17 @@
 
             &.o_current {
                 position: initial;
+
+                &.o_kanban_no_records {
+                    min-height: 64px;
```
[the remark this answers is not in the document]
Comment (res-odoo) [reply addons/web/static/src/scss/kanban_view_mobile.scss]: It's the size of the mobile tab height, i made the change code for using the SCSS variable.

**Comment by pparidans** [comment]
@robodoo r+

**Comment by pparidans** [comment]
@res-odoo This PR will need a rebase... which could require to bring back the `o_current` class if I'm not wrong!
To discuss.

**Comment by pparidans** [comment]
@robodoo r-

**Comment by pparidans** [comment]
@robodoo r+

**Comment by adr-odoo** [comment]
@robodoo retry

**Comment by adr-odoo** [comment]
@robodoo r-

**Comment by adr-odoo** [comment]
@robodoo r+

**Comment by adr-odoo** [comment]
@robodoo r-

**Comment by pparidans** [comment]
@robodoo rebase-merge

**Comment by pparidans** [comment]
@robodoo retry

**Pull request closed without merging**